AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -33,7 +33,7 @@ - (void)viewDidLoad
 {
   [super viewDidLoad];
 
-  // Test if adblock browser is installed 
+  // Test if adblock browser is installed
```

Comment:
Unrelated, but to speed things up I'll leave it out when merging this and apply it as a separate Noissue.